Classes/Private/Manager/Objects/OCTActiveFile+Variants.h

```diff
@@ -0,0 +1,58 @@
+//
+//  OCTActiveFile+Variants.h
+//  objcTox
+//
+//  Created by stal on 9/7/2015.
+//  Copyright © 2015 Zodiac Labs. All rights reserved.
+//
+
+#import "OCTSubmanagerFiles.h"
+#import "OCTActiveFile.h"
+
+#define AVERAGE_SECONDS 10
+
+@class OCTMessageFile;
+
+@interface OCTActiveFile ()
+
+@property (weak)      OCTSubmanagerFiles *fileManager;
+// Observation: I'd rather not hold on to a Realm object. Who knows what could
+//              happen.
```

Comment:
> Observation: I'd rather not hold on to a Realm object. Who knows what could happen.

Maybe it would be better to store `uniqueIdentifier` of object and get it with `[OCTRealmManager objectWithUniqueIdentifier:]` when needed?